internal/hcl/graph_vertex_module_call.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
+var moduleCallArgs = []string{"source", "version", "for_each", "count", "providers", "depends_on", "lifecycle"}
```

Comment:
can we change this to a map lookup so we don't have to loop over the list to check if the attr is a moduleCallArg

Reply:
Is your reasoning for code readability or performance? I find this more readable than using a `map[string]struct{}` type, and think any performance improvements will be negligible (if at all) for small slices.

Reply:
reasoning is for performance, but I also find map lookup more concise, eg:

```
func attrIsVarInput(name string) bool {
     _, exists := moduleCallArgs[name]
     return exists
}
```

vs 

```
func attrIsVarInput(name string) bool {
	for _, arg := range moduleCallArgs {
		if name == arg {
			return false
		}
	}

	return true
}
``

don't have a strong opinion about it so feel free to keep